### ISSUE TITLE
Support Avalonia 11 RC?

### ISSUE THREAD

**Issue by alessandrofrancesconi** [AvaloniaUtils/DialogHost.Avalonia#32]
Avalonia has reached the 11.0 RC candidate: https://github.com/AvaloniaUI/Avalonia/discussions/11593

If I upgrade the project to use Avalonia `11.0.0-rc1.1`, I get these errors:

```
1>C:\dev\DialogHost.Avalonia\DialogHost.Avalonia\DialogOverlayPopupHost.axaml.cs(88,86,88,93): error CS0122: 'IManagedPopupPositionerPopup.Screens' is inaccessible due to its protection level
1>C:\dev\DialogHost.Avalonia\DialogHost.Avalonia\DialogOverlayPopupHost.axaml.cs(97,43,97,73): error CS0122: 'IManagedPopupPositionerPopup.ParentClientAreaScreenGeometry' is inaccessible due to its protection level
1>C:\dev\DialogHost.Avalonia\DialogHost.Avalonia\DialogOverlayPopupHost.axaml.cs(114,45,114,52): error CS0122: 'IManagedPopupPositionerPopup.Scaling' is inaccessible due to its protection level
1>C:\dev\DialogHost.Avalonia\DialogHost.Avalonia\DialogOverlayPopupHost.axaml.cs(101,43,101,56): error CS0122: 'IManagedPopupPositionerPopup.MoveAndResize(Point, Size)' is inaccessible due to its protection level
```

**Comment by SKProCH** [comment]
Yep, i'll try to upgrade in the next day or two

**Comment by alessandrofrancesconi** [comment]
I really thank you for this

**Comment by aldelaro5** [comment]
Oh nice I was about to post an issue about this because I had the intention to pr it, but I hit some issues.

I raised 4 blockers I found here https://github.com/AvaloniaUI/Avalonia/discussions/11593#discussioncomment-6060170 and the reply from avalonia's maintainer seems to be that the use of these apis might not be needed anymore. I just wanted to post this in case it comes up.

**Comment by flarive** [comment]
Cool @SKProCH, also waiting for DialogHost Avalonia to be compatible with Avalonia 11 RC1 :)
Thanks a lot

**Comment by SKProCH** [comment]
Should be resolved in `0.7.4` version in nuget.

**Comment by flarive** [comment]
Works perfectly, thanks a lot !
